src/Illuminate/Support/Collection.php

```diff
@@ -327,6 +327,7 @@ public function get($key, $default = null)
      * Group an associative array by a field or using a callback.
      *
      * @param  callable|string  $groupBy
+     * @param  bool $preserveKeys
```

Comment:
We don't align these anymore, but we do separate the type from the variable name by two spaces.